gloc

```diff
@@ -1581,7 +1581,7 @@ sub get_embedded {
 
     for (keys %encoded) {
         $embedded->{$_} = do {
-            my $loader = Gtk2::Gdk::PixbufLoader->new();
+            my $loader = Gtk3::Gdk::PixbufLoader->new();
             $loader->write( decode_base64( $encoded{$_} ) );
```

Comment:
From testing, it appears that a change is needed here to the format of the argument to `write()`. I suggest:
```
my $img_data = decode_base64($encoded{$_});
$loader->write( [unpack 'C*', $img_data] );
```